Makefile

```diff
@@ -209,6 +209,38 @@ check-kuttl:
 	${PGO_KUBE_CLIENT} ${KUTTL_TEST} \
 		--config testing/kuttl/kuttl-test.yaml
 
+# KUTTL_ params -- could be left out entirely and just set as env vars
+KUTTL_PSQL_VERSION ?= 14
```

Comment:
⛏️ Higher up we use `PGO_PG_VERSION` for this. Perhaps `KUTTL_PG_VERSION` here? When I see `PSQL` I think of the client specifically.